src/wwwroot/app/views/candidates/candidates.controller.js

```diff
@@ -126,6 +131,25 @@ export default function CandidatesController(
       LocalStorageService.set('candidates', vm.candidates);
    }
 
+   function _sortBy(column) {
+      let searchPredicate = cloneDeep(vm.candidate);
+      searchPredicate.sortBy = column;
+      searchPredicate.sortAsc = (searchPredicate.sortBy === vm.candidate.sortBy) ?
+         !!(searchPredicate.sortAsc ^ true) : true; // eslint-disable-line no-bitwise
```

Comment:
make this one more understanable. 
> !!
 it's like
!!! of mine. Could be there some clear way?